src/main/java/kanbanflow/ui/pages/Board.java

```diff
@@ -12,10 +12,12 @@
 
 package kanbanflow.ui.pages;
 
+import core.selenium.WebDriverManager;
 import kanbanflow.ui.BasePage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.FindBy;
 
 /**
```

Comment:
please write the documentation in third person

Reply:
That is write in third person.